software/kernel/boards/old_revisions/controller_revA/jtag/flash-debug.jlink

```diff
@@ -1,5 +1,5 @@
 r
-loadfile target/sam4l/debug/controller.hex
+loadfile target/thumbv7em-none-eabi/debug/controller.hex
```

Comment:
Can I recommend you back out all changes in `old_revisions/`. My goal with those folders was to not push updates to them since they likely depend on things that aren't in Tock/signpost anymore and should only work when checking out the old commit when they were first created.